src/enact/interfaces.py

```diff
@@ -16,24 +16,20 @@
 
 import abc
 import functools
-import json
 from typing import (
-  Dict, Generic, Iterable, List, NamedTuple, Optional, Tuple, Type, TypeVar,
-  Union, cast)
+  Dict, Generic, Iterable, List, Optional, Tuple, Type, TypeVar,
+  Union)
 
 from enact import acyclic
+from enact import types
 
 
-JsonLeaf = Union[int, float, str, bool, None]
-Json = Union[JsonLeaf, List['Json'], Dict[str, 'Json']]
-
 
 PRIMITIVES = (int, float, str, bytes, bool, type(None))
 
 
 Primitives = Union[
-  int, float, str, bytes, bool,
-  None]
+  int, float, str, bytes, bool, None]
```

Comment:
Feels like PRIMITIVES and Primitives should go in types.py now that it exists.

Reply:
Done